Tests/GraphQL/HyperwalletTransferMethodConfigurationTests.swift

```diff
@@ -164,6 +164,23 @@ class HyperwalletTransferMethodConfigurationTests: XCTestCase {
         XCTAssertEqual(fees?.count, 1)
         XCTAssertEqual(fees?.first?.feeRateType, "FLAT")
         XCTAssertEqual(fees?.first?.value, "2.00")
+        let bankAccountIdMask = graphQlResponse?.fieldGroups()?
+            .first(where: { $0.group == "BUSINESS_INFORMATION" })?.fields?
```

Comment:
This response json seems wrong. The request is for Individual and response contains Business Fields. Can you look into it and update the correct response